lib/arel_extensions/helpers.rb

```diff
@@ -20,7 +20,8 @@ def self.column_of_via_arel_table(table_name, column_name)
   rescue NoMethodError
     nil
   rescue => e
-    warn("Warning: Unexpected exception caught while fetching column name for #{table_name}.#{column_name} in `column_of_via_arel_table`\n#{e.class}: #{e}")
```

Comment:
e itself is not useful?

Reply:
`e` was printing an inspected object, and it's mega-huge (in the problem I was resolving, it might be clearer for others, but I doubt it). The backtrace was far more informative and useful.